### PR TITLE
fix css - change green to red, according to issue

### PR DESCRIPTION
**Pull request by echolabstech**
### Background context or notes?
[issue5](https://github.com/echolabstech/opensourcetalk/issues/5)
### todo (if appropriate):
- [ ] tests were added or updated
- [ ] update knowledge base
- [ ] add new (Python) dependencies
- [ ] environment variables
- [ ] update README
- [ ] all ticket requirements are met

_____________________________________________________________________________________________________________________
### reviewer todo (if appropriate):
- [ ] at least one reviewer has seen this PR work on local
- [ ] this PR does not break tests

make a [template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) of this saved reply

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
   </head>
   <style>
     h1 {
-      color: green;
+      color: red;
```
Comment (ansufei) [inline index.html:14]: the css is good but the code is broken on line 22, please fix

**Review by ansufei — CHANGES_REQUESTED** [review]

**Comment by echolabstech** [comment]
@ansufei what's the problem? Can you show me?

**Comment by ansufei** [comment]
Did you test locally? This is what I'm seeing:
image

**Review by ansufei — APPROVED** [review]
Ok now!

**Pull request merged**